README.md

```diff
@@ -12,43 +10,125 @@ This package allows Python developers to write software that makes use of the We
 
 [Welkin Health API Documentation](https://developers.welkinhealth.com/)
 
-### Installing
+## Installing
 
 ```
 pip install welkin
 ```
 
-### Quick Start
+## Quick Start
 
 ```python
 from welkin import Client
 
-welkin = Client(
+client = Client(
```

Comment:
`client` is a really common variable name in production systems, I'd leave this example as-is.